src/events/message.js

```diff
@@ -5,35 +5,37 @@ const { allowedTags } = require('../boot');
 
 const retries = new Map();
 
-function throttleUser(
-  author,
-  callback,
-  throttleTime = config.msgThrottleTime,
-  retryThreshold = config.msgRetryThreshold,
-) {
-  if (retries.has(author.username)) {
-    const retryCount = retries.get(author.username) + 1;
-
-    retries.set(author.username, retryCount);
+function handleRetry(author, retryThreshold = config('bot.msgRetryThreshold')) {
+  const retryCount = retries.get(author.username) + 1;
 
-    if (retryCount >= retryThreshold * 2) {
-      const attachment = new Attachment(getReservedImage('angryPepe'));
+  retries.set(author.username, retryCount);
 
-      author.send('**FODA-SE PÁ, NÃO PERCEBESTE A MENSAGEM CRL?!?!**', {
-        files: [attachment],
-      });
+  if (retryCount >= retryThreshold * 2) {
+    const attachment = new Attachment(getReservedImage('angryPepe'));
 
-      return;
-    }
+    author.send('**FODA-SE PÁ, NÃO PERCEBESTE A MENSAGEM CRL?!?!**', {
```

Comment:
Probably best to merge this one first, or #24 and then come back to it, since some code overlaps.

Reply:
Yep, after merging one of them we'll probably have conflicts.